drivers/hmis/app/graphql/types/base_audit_event.rb

```diff
@@ -0,0 +1,57 @@
+###
+# Copyright 2016 - 2022 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class BaseAuditEvent < BaseObject
+    def self.create(node_class)
+      Class.new(self) do
+        graphql_name("#{node_class.graphql_name}AuditEvent")
+        field :item, node_class, null: false
+
+        define_method(:schema_type) do
+          node_class
+        end
+      end
+    end
+
+    field :id, ID, null: false
+    field :event, HmisSchema::Enums::AuditEventType, null: false
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :user, HmisSchema::User, null: true
+    field :object_changes, Types::JsonObject, null: true, description: 'Format is { field: { fieldName: "GQL field name", displayName: "Human readable name", values: [old, new] } }'
+
+    def user
+      Hmis::Hud::User.find_by(id: object.whodunnit)
```

Comment:
Pretty sure this should be `Hmis::User` , not `Hmis::Hud::User`.

https://github.com/greenriver/hmis-warehouse/blob/d1f3cb8f71a51a355fec69a61cd8bcb5f4735816/drivers/hmis/app/controllers/hmis/base_controller.rb#L39-L41

Reply:
I haven't tested this, so I'm not sure. You might happen to have the same id for your Hmis::Hud::User and your Hmis::User, which would mask this issue.

Reply:
You are correct, it should return `Hmis::User` and the type should be changed to `Application::User` rather than `HmisSchema::User`. This is done now